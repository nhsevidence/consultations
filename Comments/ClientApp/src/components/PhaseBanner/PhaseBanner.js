// @flow

import React from "react";

export const PhaseBanner = () => {
	return (
		<p className="phase-banner">
			<span className="phase-banner__tag">
				<span className="tag tag--impact tag--beta">Beta</span>
			</span>
			<span className="phase-banner__label">
				NICE Consultations is in development. This means it is not feature
				complete and there may be issues.
				<a
					href="https://github.com/nhsevidence/consultations"
					rel="noopener noreferrer"
					target="_blank"
				>
					Visit this page if you wish to report an issue.
				</a>
			</span>
		</p>
	);
};
