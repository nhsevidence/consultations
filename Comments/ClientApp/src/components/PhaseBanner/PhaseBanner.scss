.tag {
  &--agile {
    background-color: #007AB8;
  }
}
