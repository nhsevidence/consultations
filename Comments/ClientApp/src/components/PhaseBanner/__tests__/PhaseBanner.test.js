import React from "react";
import { shallow } from "enzyme";

import { PhaseBanner } from "../PhaseBanner";

describe("[ClientApp] ", () => {
	describe("PhaseBanner ", () => {
		it("Renders with correct link", () => {
			const wrapper = shallow(
				<PhaseBanner />
			);
			expect(wrapper.find("p span a").prop("href")).toEqual(
				"https://github.com/nhsevidence/consultations"
			);
		});
	});
});
