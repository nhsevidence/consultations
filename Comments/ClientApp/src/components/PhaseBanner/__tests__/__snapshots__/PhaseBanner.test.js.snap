// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[ClientApp]  PhaseBanner  Matches snapshot 1`] = `
<p
  className="phase-banner"
>
  <span
    className="phase-banner__tag"
  >
    <span
      className="tag tag--impact tag--delta"
    >
      delta
    </span>
  </span>
  <span
    className="phase-banner__label"
  >
    My Project
     is in development. This means it is not feature complete and there may be issues.
     
    <a
      href="http://mygitrepo.com"
      rel="noopener noreferrer"
      target="_blank"
    >
      Visit this page if you wish to report an issue.
    </a>
  </span>
</p>
`;
